pkg/openstack/rabbitmq.go

```diff
@@ -389,14 +389,27 @@ func reconcileRabbitMQ(
 					map[string]string{networkv1.AnnotationHostnameKey: hostname})
 		}
 
+		ipAddressToBind := "0.0.0.0"
+		if IPv6Enabled {
+			ipAddressToBind = "::"
```

Comment:
I expect you can just set it to `::`. This usually works for both IPv4 and IPv6

Reply:
Thanks for the suggestion. I'll try.

Reply:
Thanks again. This allowed me to greatly simplify the PR. I also found out, that the `x.tcp.ip` can always be set (even with TLS enabled), which allowed for further simplification.